dockstore-integration-testing/src/test/java/io/dockstore/webservice/NotebookIT.java

```diff
@@ -219,6 +220,30 @@ void testWithImage() {
         assertEquals("quay.io/seqware/seqware_full/1.1", notebook.getWorkflowVersions().get(0).getKernelImagePath());
     }
 
+    @Test
+    void testSnapshot() {
+        CommonTestUtilities.cleanStatePrivate2(SUPPORT, false, testingPostgres);
+        ApiClient apiClient = getOpenAPIWebClient(BasicIT.USER_2_USERNAME, testingPostgres);
+        WorkflowsApi workflowsApi = new WorkflowsApi(apiClient);
+        workflowsApi.handleGitHubRelease("refs/tags/with-tagged-kernel-v1", installationId, simpleRepo, BasicIT.USER_2_USERNAME);
+        Workflow notebook = workflowsApi.getWorkflowByPath(simpleRepoPath, WorkflowSubClass.NOTEBOOK, "versions");
+        WorkflowVersion version = notebook.getWorkflowVersions().stream().filter(WorkflowVersion::isValid).findFirst().get();
+        // Publish the notebook
+        PublishRequest publishRequest = new PublishRequest();
+        publishRequest.setPublish(true);
+        workflowsApi.publish1(notebook.getId(), publishRequest);
+        assertFalse(version.isFrozen());
+        assertEquals(0, testingPostgres.runSelectStatement("select count(*) from entry_version_image where versionid = " + version.getId(), long.class));
+        // Snapshot the notebook
+        version.setFrozen(true);
```

Comment:
To be clear, it looks like snapshotting was working previously, it just wouldn't snapshot the images as well? 
i.e. was snapshotting failing with a 500 or anything? This  doesn't seem to change anything in that regard

Reply:
> To be clear, it looks like snapshotting was working previously, it just wouldn't snapshot the images as well? i.e. was snapshotting failing with a 500 or anything? This doesn't seem to change anything in that regard

Yup, previously, it would snapshot the notebook without error, but would not include the kernel image.